corehq/apps/app_manager/static/app_manager/js/apps_base.js

```diff
@@ -19,4 +19,16 @@ $(function () {
     if (v2) {
         hqLayout.utils.setIsAppbuilderResizing(true);
     }
+
+    $(document).ajaxComplete(function(e, xhr, options) {
+        if (/edit_form_attr/.test(options.url) ||
```

Comment:
Oh, I see. Seems like a question then of whether to store the set of "saving" urls or the set of "non-saving" urls (at least the version check url).